.github/workflows/examples-on-commit.yml

```diff
@@ -0,0 +1,39 @@
+---
+name: Test examples on commit
+
+on:
+  push:
+    paths:
+      - glass/**
+      - examples/**
+    branches:
+      - main
+  pull_request:
+    paths:
+      - glass/**
+      - examples/**
+    types:
+      - opened
+      - ready_for_review
+      - reopened
+      - synchronize
+
+concurrency:
+  cancel-in-progress: true
+  group: >-
+    ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+
+jobs:
+  test-examples:
+    if: github.event.pull_request.draft == false
+    name: Test examples
```

Comment:
Do we want one of these files or `example-on-label` to have a different name? Feel like it could get confusing.